### PR TITLE
feat(inputs.sql): Add support for IBM Netezza

### PR DESCRIPTION
**Pull request by mplpl**
# Required for all PRs

- [x] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

Added support for IBM Netezza data warehouse to inputs/sql plugin.

### CODE REVIEW

File: go.mod
Comment (powersj) [inline go.mod:221]: Can you please move this down into the other indirect section.

**Review by powersj — COMMENTED** [review]
Hi,

Thanks for the PR. One small comment below, however the 32-bit build failures are the larger concern. I see you commented on https://github.com/IBM/nzgo/issues/38

File: go.mod
Comment (mplpl) [reply go.mod]: Done

**Comment by powersj** [comment]
What is your plan to resolve the 32-bit build issues?

**Comment by mplpl** [comment]
I'm going to talk to IBM on Monday about that. It may take some time for this issue to become fixed.

**Comment by srebhan** [comment]
Just as a hint: As workaround you could guard the driver with build-tags excluding 32-bit archs (similar to sqlite)...

**Comment by mplpl** [comment]
Thanks for suggestion @srebhan 
In the meanwhile I've opened a PR to nzgo to support 32-bit environments. Let's see how it goes (I'm in touch with the nzgo maintainer). If it does not, I'll go sqlite way.

**Comment by powersj** [comment]
@mplpl,

Looks like https://github.com/IBM/nzgo/pull/60 was landed. Is an update to the version imported all that is required?

File: go.mod
Comment (Hipska) [inline go.mod:247]: So you are adding v12 and v11? I think all v11 lines in go.mod and go.sum can be removed?

Comment (mplpl) [reply go.mod:247]: Thanks - I'll fix that

Comment (mplpl) [reply go.mod:247]: To be clear, this is an indirect dependency from github.com/IBM/nzgo/v12 and so to remove it from here, I needs to be removed from nzgo/v12 first. I'll open a PR for nzgo with such a change.

**Comment by mplpl** [comment]
@powersj My PR with support for 32-bit envs for nzgo was indeed merged, but a new version of this driver has not yet been released. I'm in touch with the maintainers and waiting for their decision about when to release it.

**Comment by powersj** [comment]
> github.com/IBM/nzgo

They haven't seem to have tagged a release in 2 years. Our next feature release is in a few weeks and I would like to see this land in that, instead of waiting till March.

What we can do is import the repo at a specific commit, rather than a specific tag/release. Then if/when they do an actual release we can switch the go.mod entry to a commit hash.

**Comment by powersj** [comment]
@mplpl I have checked out your branch, rebased on our master, updated it to use nzgo with the master branch from today, and run go mod tidy.

**Comment by mplpl** [comment]
@powersj ok, thanks. It is Diwali holiday in India, and the person responsible for nzgo is away now. But she should be back in a few days and promised to update me with when they plan to tag a new version.

**Review by Hipska — APPROVED** [review]
Is it possible to add integration test like we have for others? Or would that not be easy to achieve?

**Comment by srebhan** [comment]
@Hipska I don't see any official container image, so having an integration test would be hard...

**Review by srebhan — APPROVED** [review]
Thanks for your contribution @mplpl! Feel free to bump the driver once a new release is out.

**Pull request merged**
